### PR TITLE
Remove collab support

### PR DESCRIPTION
**Pull request by jtanx**
This removes collab support. Related issues: #3512.

@skef, you mentioned that you'd like to keep some of it, would you mind pointing out exactly which parts you were interested in? (It was easier to remove everything at once - then we can go back and add what we want)

### CODE REVIEW

**Comment by frank-trampe** [comment]
Maybe a designated coded comment indicating a state change?

**Comment by skef** [comment]
@frank-trampe That's what I was thinking; I just don't know if I can really defend the effort. The definitive guide to the collab project as it wound up is right here in the single commit to remove it.

**Comment by skef** [comment]
This is a little cute sounding, but what if we add a branch that reverses the effect of this commit as a single commit? Basically a "this is all of the collab project at the time it was removed, rebased onto the master of that time" branch.

**Comment by jtanx** [comment]
I think it's very easy for that to get lost in a branch. What about this? Create a boneyard repo that documents previously removed features, and links to the commits that removed it/what would be involved in resurrecting it.

**Comment by ctrlcctrlv** [comment]
As an erstwhile git consultant, I approve

**Comment by skef** [comment]
@jtanx even better. 
 
I guess a different way of putting my point is that in reviewing this commit, I found that **it** is what I would actually want most if I were to try to resurrect part of the project. Not some pieces but the whole thing, in some kind of easily browsable form.

**Comment by frank-trampe** [comment]
This appears to be a case where it is actually possible to please everybody.

File: fontforgeexe/charview.c [hunk 1]
```diff
@@ -5014,32 +5013,8 @@ return;		/* I treat this more like a modifier key change than a button press */
       case cvt_pointer:
```
[the remark this answers is not in the document]
Comment (jtanx) [reply fontforgeexe/charview.c]: Good spot - fixed

**Comment by frank-trampe** [comment]
Hmm. I sure can't think of one. I'll ask on the list just to be safe.

**Comment by jtanx** [comment]
fwiw I've documented what I know here: https://github.com/fontforge/fontforge/wiki/Defunct-or-Removed-features-from-FontForge

**Comment by frank-trampe** [comment]
@jtanx, good work! It's amazing how many skeletons we've managed to move from the closet to the crypt.

**Comment by jtanx** [comment]
On that cli flag, I'm going to take the lack of response and the fact that it would only have worked on gxdraw as "no it's not used"

**Pull request merged**